composer.json

```diff
@@ -5,55 +5,54 @@
         "contao",
         "event"
     ],
-    "type": "contao-module",
+    "type": "contao-bundle",
     "homepage": "http://c-c-a.org/",
     "license": "LGPL-3.0+",
     "authors": [
         {
-            "name": "Tristan Lins",
-            "email": "tristan.lins@bit3.de",
-            "homepage": "http://bit3.de",
+            "name": "Christian Schiffler",
+            "email": "c.schiffler@cyberspectrum.de",
+            "homepage": "http://www.cyberspectrum.de",
+            "role": "Developer"
+        },
+        {
+            "name": "Stefan Heimes",
+            "email": "stefan_heimes@hotmail.com",
             "role": "Developer"
         }
     ],
     "support": {
-        "email": "info@bit3.de",
+        "email": "info@c-c-a.org",
         "issues": "https://github.com/contao-community-alliance/event-dispatcher/issues",
         "wiki": "https://github.com/contao-community-alliance/event-dispatcher/wiki",
         "irc": "irc://irc.freenode.org/contao.dev",
         "source": "https://github.com/contao-community-alliance/event-dispatcher"
     },
     "require": {
-        "php": ">=5.3",
-        "contao/core": ">=2.11,<4-dev",
-        "contao-community-alliance/composer-plugin": "~2.0",
-        "contao-community-alliance/dependency-container": "~1.6",
-        "symfony/event-dispatcher": "~2.3"
+        "php": ">=5.6",
+        "contao/core-bundle": "~4.4",
+        "contao-community-alliance/dependency-container": "~2.0@dev",
```

Comment:
It is used by the `src/Resources/contao/services.php` which starts the whole thing up.